django_mongodb/schema.py

```diff
@@ -60,21 +96,97 @@ def remove_field(self, model, field):
         # Unset field on existing documents.
         if column := field.column:
             self.connection.database[model._meta.db_table].update_many({}, {"$unset": {column: ""}})
+            if self._field_should_be_indexed(model, field):
+                self._remove_field_index(model, field)
 
     def alter_index_together(self, model, old_index_together, new_index_together):
-        pass
+        olds = {tuple(fields) for fields in old_index_together}
+        news = {tuple(fields) for fields in new_index_together}
+        # Deleted indexes
+        for field_names in olds.difference(news):
+            self._remove_composed_index(model, field_names, {"index": True, "unique": False})
+        # Created indexes
+        for field_names in news.difference(olds):
+            self._add_composed_index(model, field_names)
 
     def alter_unique_together(self, model, old_unique_together, new_unique_together):
         pass
 
-    def add_index(self, model, index):
-        pass
+    def add_index(self, model, index, field=None):
+        if index.contains_expressions:
+            return
```

Comment:
Should we not raise a `NotSupportedError` or what's the tactic for handling indexes with expressions. UUIC these are complex types that exist beyond the standard ASC/DESC indexing capabilities of MongoDB.

Reply:
They are ignored (in case the index is useful on another database backend) as per https://docs.djangoproject.com/en/dev/ref/models/indexes/#django.db.models.Index.expressions.